main/com.tunein/dataset.tt

```diff
@@ -0,0 +1,25 @@
+dataset @com.tunein language "en" {
+    query (p_query :String)  := @com.tunein.search_channel(query=p_query, count=p_count)
+    #_[utterances=["${p_query:const} channels on the radio",
+                   "radio channels named ${p_query}",
+                   "${p_query:const} radio channel",
+                   "first ${p_count:const} radio channels matching ${p_query:const}",
+                   "top ${p_count:const} radio channels matching ${p_query:const}"]];
+
+    stream  := monitor (@com.tunein.search_channel(query=p_query))
+    #_[utterances=["when there is a new track from the ${p_query:const} channel"]];
```

Comment:
The ThingTalk you wrote means "when there is a new radio station", which is not the same.

I'm not sure we can easily do "when there is a new track". Maybe:
```
monitor (now_playing of (@com.tunein.search_channel(), id =~ p_name))
```
(which would be executed as
```
monitor (now_playing of ((@com.tunein.search_channel(), id =~ p_name)[1]))
```
so it only monitors the first result matching the given name)

Then again, you set a very high polling interval so that won't work anyway...